packages/sdk/shell/src/panels/IdentityPanel/steps/IdentityActionChooser.tsx

```diff
@@ -24,7 +25,7 @@ export const IdentityActionChooser = (props: IdentityPanelStepProps) => {
   const onInvitationEvent = useCallback((invitation: Invitation) => {
     const invitationCode = InvitationEncoder.encode(invitation);
     if (invitation.state === Invitation.State.CONNECTING) {
-      console.log(JSON.stringify({ invitationCode, authCode: invitation.authCode }));
+      log.info(JSON.stringify({ invitationCode, authCode: invitation.authCode }));
```

Comment:
make sure the composer e2e tests still pass w/ this change, iirc they depend on this log and I can't remember if info logs get omitted in the production bundle or not